src/app/scripts/cac/control/cac-control-sidebar-directions.js

```diff
@@ -93,6 +102,9 @@ CAC.Control.SidebarDirections = (function ($, Control, BikeOptions, Geocoder, Ma
 
         setFromUserPreferences();
         changeMode();
+
+        // Respond to changes on all direction input fields
+        $(options.selectors.directionInput).on('input change', planTrip);
```

Comment:
What happens when the mode is changed?  As it already has its own handler, I think it would be better to remove the new input class from it, and instead call `planTrip` from the end of its handler.  (Same with isochrone controller.) I imagine it works due to the debounce timeout letting the mode change handler finish first, but better to be explicit with that.